### ISSUE TITLE
New Pages for the Block themes

### ISSUE THREAD

**Issue by TeBenachi** [WordPress/Documentation-Issue-Tracker#108]
As we are moving forward with merging with the [Theme handbook](https://developer.wordpress.org/themes/), here is a list of suggested new pages related to the block theme. The new pages will go under the [Block theme section](https://developer.wordpress.org/themes/block-themes/). 

== 

Introduction and overview
Difference between block theme and classic theme

How to create a block theme
- File structure
- Templates
- Parts
- Ways to create the templates (Manually, Site Editor, Template Editing mode)
- Exporting
- Additional templates

Global Styles and Theme.json
Theme Support

Converting classic theme to block theme (#106)
- Benefits of changing from theme to block theme
- Converting customizer into blocks
- Using global styles
- Examples (Site logo, Colors or widgets) including snippets and code
- Theme before and after
- Links and resources

What is patterns?
How to make patterns?
What are the patterns category?

**Comment by carolinan** [comment]
In addition to the above, we need to include information about block markup and how to use it.

Please consider the following updates to be a starting point.
**Feel free to edit/remove/add content, including moving it to different documents**

There is a scheduled update for the entry page for block themes that is ready to be reviewed.
I listed the [theme basics chapter](https://developer.wordpress.org/themes/basics/) as a prerequisite so that we do not need to bring up details about functions.php, style.css, etc a second time.
https://developer.wordpress.org/wp-admin/post.php?post=138837&action=edit

I created a new one-page draft that so far includes:
- File structure
- How to create templates and template parts in the site editor,
- Exporting.

I left the "creating templates in the template editor section" as a link to the existing support article.
This is not intended to be published as-is, but to be moved to new pages if we decided it needs to be separate pages and just sections.
https://developer.wordpress.org/wp-admin/post.php?post=138840&action=edit

**Comment by carolinan** [comment]
Theme.json can be used by both classic themes and block themes. The existing page is in the advanced section, and has a scheduled update that is ready for review:
https://developer.wordpress.org/wp-admin/post.php?post=138713&action=edit

**Comment by carolinan** [comment]
Added a section about enqueueing block styles, but it is very long. Eventually, this may be replaced by a proper developer reference page for the new wp_enqueue_block_style().

**Comment by carolinan** [comment]
Added a theme.json setting and add_theme_support comparison table.

**Comment by carolinan** [comment]
Theme authors are asking how to correctly load JavaScript in the different editors.
I think we need to include that, but I don't actually have the answer to that.
(My personal preference is don't do that, and rely on blocks instead)

**Comment by carolinan** [comment]
Yes, we can refer to the Loading JavaScript page in the block editor handbook.
I was hesitant to recommend enqueue_block_editor_assets because it used to trigger a warning. The warning has now been removed.

**Comment by carolinan** [comment]
The original page I started writing in is now called "Block theme setup"
https://developer.wordpress.org/wp-admin/post.php?post=138840&action=edit

I move the template/parts and block markup content to a new page called "Templates and template parts"
https://developer.wordpress.org/wp-admin/post.php?post=139037&action=edit

**Comment by carolinan** [comment]
At the start of the block markup section, it says:
> You do not create templates and template parts using standard HTML. Instead, they contain block markup that represents
> blocks and block settings.

This reads rude to me and needs to be softened up. I was trying to avoid using a passive voice.

**Comment by carolinan** [comment]
Most of these pages have already been added; the ones I am unsure of is:

1. Examples (Site logo, Colors or widgets), including snippets and code
2. Theme before and after

I am not sure what 1 refers to. I think 2 might be the comparison table?
https://developer.wordpress.org/themes/block-themes/#differences-and-similarities-between-classic-themes-and-block-themes
